modules/getting-started/pages/index.adoc

```diff
@@ -5,35 +5,87 @@ endif::[]
 :keywords: getting started, how to, platform, mule, api
 :noindex:
 
```

Comment:
This is just a copy of the previous page, needed for architectural reasons. Please don't edit this one--edit the other file and I'll copy any changes here.